addon-test-support/scroll-into-view-mock.js

```diff
@@ -0,0 +1,31 @@
+import sinon from 'sinon';
+import { find } from '@ember/test-helpers';
+export default function setupScrollIntoView(hooks) {
+  hooks.beforeEach(function () {
+    this.scrollIntoViewSandbox = sinon.createSandbox();
+    const scrollIntoViewMock = this.scrollIntoViewSandbox.stub();
```

Comment:
instead of `stub`, I think you can do `sinon.fake`?

https://sinonjs.org/releases/latest/fakes/